dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -414,6 +414,25 @@ public boolean terminateUser(
         return true;
     }
 
+    @PATCH
+    @Timed
+    @UnitOfWork
+    @Path("/user/{userId}")
+    @RolesAllowed("admin")
+    @Operation(operationId = "reactivateUser", description = "Reactivate a user that has been terminated", security = @SecurityRequirement(name = OPENAPI_JWT_SECURITY_DEFINITION_NAME))
```

Comment:
"...has been terminated" -> "...has been banned"? Not sure of our exact terminology.